### PR TITLE
Update Semaphore configuration

### PR DESCRIPTION
**Pull request by kamran-github**

### CODE REVIEW

**Comment by kamran-github** [comment]
Semaphore added here (Kamran)

**Comment by kamran-github** [comment]
Update Semaphore configuration

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    os_image: macos-xcode11
+blocks:
+  - name: Tests
+    task:
+      jobs:
+        - name: swift test
+          commands:
+            - checkout
+            - swift build -v
+            - swift test -v
```
Comment (kamran-github) [inline .semaphore/semaphore.yml:15]: Update Semaphore configuration

**Comment by KamranTNK** [comment]
Approve by KAMRANTNK11

**Pull request closed without merging**